spinalcordtoolbox/reports/qc2.py

```diff
@@ -661,19 +762,22 @@ def mosaic(img: Image, centers: np.ndarray, radius: tuple[int, int] = (15, 15),
     return np.block([cropped[i:i+num_col] for i in range(0, len(cropped), num_col)])
 
 
-def add_orientation_labels(ax: mpl_axes.Axes, radius: tuple[int, int] = (15, 15)):
+def add_orientation_labels(ax: mpl_axes.Axes, radius: tuple[int, int] = (15, 15), orientation: str = 'SAL'):
     """
     Add orientation labels (A, P, L, R) to a figure, yellow with a black outline.
     """
     # Ensure that letter locations are determined as a function of the bounding box. For a 15,15 radius (30x30):
     #    A                    [12,  6]
     # L     R   -->  [0, 17]            [24, 17]
     #    P                    [12, 28]
+    letters = ['A', 'P', 'L', 'R']
+    if orientation == 'RSP':
+        letters = ['S', 'I', 'P', 'A']
```

Comment:
I think instead of mapping `orientation` to `letters`, it might make more sense to instead just put `letters` as the parameter.

That way, it's clear at the caller which letters are being used.

Reply:
Okay ! Makes sense ! On it now !